tests/tck/features/match/SeekByTag.feature

```diff
@@ -18,9 +18,7 @@ Feature: Match seek by tag
       MATCH (v:bachelor)
       RETURN id(v) AS vid, v.age AS age
       """
-    Then the result should be, in any order:
-      | vid          | age |
-      | 'Tim Duncan' | 42  |
+    Then a SemanticError should be raised at runtime: Not found prop `age'
```

Comment:
`(v:bachelor)` doesn't mean vertex `v` only has one tag named `bachelor`. It means `v` must have the tag `bachelor` and maybe have multiple other tags.

Reply:
So this limitation check is not a reasonable way.